packages/sessions-sdk-react-native/scripts/setup-deps.js

```diff
@@ -0,0 +1,788 @@
+#!/usr/bin/env node
```

Comment:
What's the point of this script?  It appears to be some stuff related to initial repo scaffolding, is there any reason to have this script in the repo?  Are we ever expecting to run this again?